README.md

```diff
@@ -146,7 +146,46 @@ This allows you to use `eslint` to look for bugs and/or bad practices, and use
 
 `prettier-eslint` will **only** propagate _parsing_ errors when either `prettier` or `eslint` fails. In addition to propagating the errors, it will also log a specific message indicating what it was doing at the time of the failure.
 
-**Note:** `prettier-eslint` will not show any message regarding broken rules in either `prettier` or `eslint`.
+**Note:** `format` will not show any message regarding broken rules in either `prettier` or `eslint`.
+
+## Capturing eslint messages
```

Comment:
```suggestion
## Capturing ESLint messages
```

Reply:
On the first suggestion, I didn't change that occurrence because neither 'format' nor 'analyze' _show_ any messages (as in write them anywhere), just 'analyze' will pass them back. So the sentence seemed to still be correct. But I will be happy to change it if you like, just let me know. And I will change the second one once I hear about that, thanks.

Reply:
@gwhitney It's only about changing `eslint` to `ESLint` as that is the name proper name not just the package name.

Reply:
No, I get that but what about the first change you suggested of  "prettier-eslint" to "format"? I thought about that when I first submitted the PR but since neither format or analyze "shows" anything, i.e., they don't write any messages to the console or anything, I left it the way it was because it still seemed accurate. But I will change it to 'format' if you like. Please just let me know. Thanks.

Reply:
@gwhitney Oh I see. I didn't suggest that. Not sure why it's showing up as a change.